README.md

```diff
@@ -62,28 +62,26 @@ http {
 And here's minimal viable config for 4 locations you need to set up. These locations are described in the sections below:
 ```
 # video location
-location ~ ^/(?<luamp_flags>([0-9a-zA-Z_,\.:]+)\/|)(?<luamp_filename>[0-9a-zA-Z_\-\.]+\.mp4)$ {
```

Comment:
you mean keep filename instead of public_id+extension? why? im asking because public_id its needed anyway now, its mandatory parameter. the choice is between get every mandatory parameter here or split string in the code.

Reply:
According to wiki filename=name+extension. So it could be confusing. We need some name for identification and this shouldnt be `filename`, `name` or `public_id`. Changed it to `media_id`.